lib/fika/compiler/parser/function_def.ex

```diff
@@ -63,12 +79,49 @@ defmodule Fika.Compiler.Parser.FunctionDef do
     |> wrap(exps)
     |> concat(require_space)
     |> ignore(string("end"))
-    |> label("function definition")
-    |> Helper.to_ast(:function_def)
+    |> label("public function definition")
+    |> Helper.to_ast(:public_function_def)
 
-  function_defs =
+  ext_atom =
+    ignore(string("\""))
+    |> repeat(choice([string("\\\""), utf8_char(not: ?")]))
+    |> ignore(string("\""))
+    |> Helper.to_ast(:ext_atom)
```

Comment:
Would this work?
```suggestion
    |> reduce({List, :to_atom, []})
```

Reply:
PS: I tested this locally and all tests passed

Reply:
Good idea!